src/widget.tsx

```diff
@@ -116,10 +116,10 @@ const KernelManagerComponent = (): JSX.Element => {
    */
 
   useEffect(() => {
-    const url = "http://localhost:8888/ks";
+    const url = "./ks";
```

Comment:
ha ! Thanks ! I was using `document.location.origin+...` locally. I'm guessing this does not work if there is a prefix (like in hub), but ill be fixed once we actually use jlab services ?

Reply:
The correct fix will be for my next PR. I just wanted to get something a bit less harcoded. I will ping you on my next commit.

Reply:
Ok, no worries, cause just this one seem broken for me, and seem to fetch the URL for /lab, 

so maybe you meant `"/ks"` not `"./ks"` ?